astropy/coordinates/angles.py

```diff
@@ -471,7 +472,18 @@ def __str__(self):
         return str(self.to_string())
 
     def _repr_latex_(self):
-        return str(self.to_string(format='latex'))
+        if self.isscalar:
+            return self.to_string(format='latex')
+        else:
+            # need to do a magic incantation to convert to str.  Regular str
+            # or array2string causes all baslashes to get doubled
```

Comment:
typo: baslashes ->  backslashes (and be PEP8 compliant: Capitalize first word and end with a full stop).